modules/assignment/src/main/java/com/intuit/wasabi/assignment/impl/AssignmentsImpl.java

```diff
@@ -1095,7 +1104,25 @@ public boolean doSegmentTest(Application.Name applicationName, Experiment.Label
         return queueLengthMap;
     }
 
-
+    @Override
+    public Map<String, Object> queuesDetails() {
+        Map<String, Object> queueLengthMap = new HashMap<String, Object>();
```

Comment:
Ok but I don't think it is necessary because the code is very simple. It just return a Map.